clients/aws-sdk-sagemaker-runtime-http2/pyproject.toml

```diff
@@ -0,0 +1,73 @@
+# Code generated by smithy-python-codegen DO NOT EDIT.
+
+
+[project]
+name = "aws_sdk_sagemaker_runtime_http2"
+version = "0.1.0"
```

Comment:
We need to talk to Jonathan if this should be `0.2.0`. I think it does since we're updating `smithy-core`.